pgx/src/datum/from.rs

```diff
@@ -408,3 +393,34 @@ impl<T> FromDatum for PgBox<T, AllocatedByPostgres> {
         })
     }
 }
+
+impl<T> FromDatum for Option<T>
+where
+    T: FromDatum,
+{
+    #[inline]
+    unsafe fn from_datum(datum: pg_sys::Datum, is_null: bool) -> Option<Self> {
+        match is_null {
+            true => None,
+            false => Some(T::from_datum(datum, is_null)),
+        }
+    }
+
+    unsafe fn from_datum_in_memory_context(
```

Comment:
I don't see a need to implement this -- the default implementation should suffice.  Also, this is basically what the default implementation is anyways.

Reply:
iirc we added this when trying to debug an ownership issue. Let me yank it! Good catch.

Reply:
Works fine with it removed!